parts/motor.go

```diff
@@ -0,0 +1,40 @@
+package parts
+
+import "fmt"
+
+type Motor struct {
+	On  bool
+	Rpm int
+}
+
+func (m *Motor) SetOn(on bool) {
+	m.On = on
+}
+
+func (m *Motor) IsOn() bool {
+	return m.On
+}
+
+func (m *Motor) SetRPM(rpm int, sb SeatbeltManipulator) error {
+	if rpm < 0 {
+		return fmt.Errorf("Negative RPM")
+	}
+	if rpm > 0 && sb.GetState() == SeatbeltStateEngauged {
```

Comment:
Nice.
Though, it would have been nice to see a seatbelt lock outside of the motor itself.